apps/wpcom-block-editor/src/default/features/rich-text.js

```diff
@@ -75,9 +76,11 @@ const ConnectedRichTextJustifyButton = compose(
 	ifCondition( props => 'core/paragraph' === props.blockName )
 )( RichTextJustifyButton );
 
-registerFormatType( 'wpcom/justify', {
-	title: wpcomGutenberg.richTextToolbar.justify,
-	tagName: 'p',
-	className: null,
-	edit: ConnectedRichTextJustifyButton,
+domReady( function() {
+	registerFormatType( 'wpcom/justify', {
```

Comment:
This is to ensure that this registration runs after the Filter has changed the `inserter` property for the coblocks/buttons?

I was initially wondering if waiting until `domReady` to register may have any adverse side effects, but I'm not really noticing any in testing so it seems to be working well.

Reply:
D37423-code has more context:

> If order for the filter to run at the correct time, we also need to enqueue the script in the header rather than the footer, so CoBlocks are registered afterwards.

>  [...] just realized that the justify format is missing when this patch is applied, which is likely caused by enqueuing the scripts on the head rather than on the footer. I'll investigate if we can register the justify format in a way that can be loaded before the editor scripts but executed afterwards.

> `domReady` did the trick: https://github.com/Automattic/wp-calypso/pull/38724/commits/4c3647a7a70fbaf356adea844578f2b95f49ca88